### PR TITLE
Unity SDK Generation

### PR DESCRIPTION
**Pull request by nan2iz**
**Assistant V1**
`alt_text` property added to Image response type
​
**Assistant V2**
`session_start_time` and state properties added to `MessageContextGlobalSystem` model
`alt_text` property added to Image response type
 location property of `RuntimeEntity` model changed from required to optional
​
**Discovery V2**
Enum update for `CreateProjectConstants`
​
**NLU**
Fix for `listClassificationsModels` through return type change and removal of unnecessary model
(note: I did not see the change on the diff)
​
**STT**
More languages supported for next generation models
(note: I did not see the change on the diff)
​
**TTS**
New voice models added
​
**CC, Disco V1, LT, NLC, PI, TA, VisRec V3 & V4**
No changes

### CODE REVIEW

File: Scripts/Services/Discovery/V1/Model/QueryAggregation.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ namespace IBM.Watson.Discovery.V1.Model
     /// <summary>
     /// An aggregation produced by  Discovery to analyze the input provided.
     /// </summary>
-    [JsonConverter(typeof(JsonSubtypes), "Type")]
+    [JsonConverter(typeof(JsonSubtypes), "type")]
```
Comment (nan2iz) [inline Scripts/Services/Discovery/V1/Model/QueryAggregation.cs:27]: I'm not so sure if I should include this change or not. I just added it here. Let me know if we need to convert it back

Comment (kevinkowa) [reply Scripts/Services/Discovery/V1/Model/QueryAggregation.cs:27]: correct, it should be lower case 👍

**Review by kevinkowa — APPROVED** [review]
LGTM! Lets wait for any news regarding the issues that other SDKs ran into before merging

File: Scripts/Services/Discovery/V2/Model/ProjectDetails.cs [hunk 2]
```diff
@@ -34,22 +40,32 @@ public class TypeValue
             /// </summary>
             public const string DOCUMENT_RETRIEVAL = "document_retrieval";
             /// <summary>
-            /// Constant ANSWER_RETRIEVAL for answer_retrieval
+            /// Constant CONVERSATIONAL_SEARCH for conversational_search
             /// </summary>
-            public const string ANSWER_RETRIEVAL = "answer_retrieval";
+            public const string CONVERSATIONAL_SEARCH = "conversational_search";
```
Comment (mediumTaj) [inline Scripts/Services/Discovery/V2/Model/ProjectDetails.cs:45]: just want to make sure this is expected

File: Scripts/Services/Discovery/V2/Model/ProjectListDetails.cs [hunk 3]
```diff
@@ -34,22 +40,32 @@ public class TypeValue
             /// </summary>
             public const string DOCUMENT_RETRIEVAL = "document_retrieval";
             /// <summary>
-            /// Constant ANSWER_RETRIEVAL for answer_retrieval
+            /// Constant CONVERSATIONAL_SEARCH for conversational_search
             /// </summary>
-            public const string ANSWER_RETRIEVAL = "answer_retrieval";
+            public const string CONVERSATIONAL_SEARCH = "conversational_search";
```
Comment (mediumTaj) [inline Scripts/Services/Discovery/V2/Model/ProjectListDetails.cs:45]: just want to make sure this is expected

File: Scripts/Services/NaturalLanguageUnderstanding/V1/NaturalLanguageUnderstandingService.cs [hunk 4]
```diff
@@ -1424,18 +1424,18 @@ private void OnCreateClassificationsModelResponse(RESTConnector.Request req, RES
...
         /// List classifications models.
         ///
-        /// (Beta) Returns all custom classifications models associated with this service instance.
+        /// Returns all custom classifications models associated with this service instance.
         /// </summary>
         /// <param name="callback">The callback function that is invoked when the operation completes.</param>
-        /// <returns><see cref="ListClassificationsModelsResponse" />ListClassificationsModelsResponse</returns>
-        public bool ListClassificationsModels(Callback<ListClassificationsModelsResponse> callback)
+        /// <returns><see cref="ClassificationsModelList" />ClassificationsModelList</returns>
+        public bool ListClassificationsModels(Callback<ClassificationsModelList> callback)
```
Comment (mediumTaj) [inline Scripts/Services/NaturalLanguageUnderstanding/V1/NaturalLanguageUnderstandingService.cs:1431]: is this breaking? response model changed names

**Review by mediumTaj — COMMENTED** [review]
👍 looks good overall, please see comments for possible breaking changes

File: Scripts/Services/NaturalLanguageUnderstanding/V1/NaturalLanguageUnderstandingService.cs (hunk 4 above)
Comment (kevinkowa) [reply Scripts/Services/NaturalLanguageUnderstanding/V1/NaturalLanguageUnderstandingService.cs:1431]: that's the fix for the NLU issue. This function was not working before

File: Scripts/Services/Discovery/V2/Model/ProjectDetails.cs (hunk 2 above)
Comment (nan2iz) [reply Scripts/Services/Discovery/V2/Model/ProjectDetails.cs:45]: Yes, this is the new enum for Discovery

File: Scripts/Services/Discovery/V2/Model/ProjectListDetails.cs (hunk 3 above)
Comment (nan2iz) [reply Scripts/Services/Discovery/V2/Model/ProjectListDetails.cs:45]: Yes, this is the new enum for Discovery

File: Scripts/Services/Assistant/V2/Model/RuntimeResponseGeneric.cs [hunk 5]
```diff
@@ -148,6 +153,11 @@ public class PreferenceValue
         [JsonProperty("agent_unavailable", NullValueHandling = NullValueHandling.Ignore)]
         public AgentAvailabilityMessage AgentUnavailable { get; protected set; }
         /// <summary>
+        /// Routing or other contextual information to be used by target service desk systems.
+        /// </summary>
+        [JsonProperty("transfer_info", NullValueHandling = NullValueHandling.Ignore)]
+        public DialogNodeOutputConnectToAgentTransferInfo TransferInfo { get; protected set; }
+        /// <summary>
```
Comment (nan2iz) [inline Scripts/Services/Assistant/V2/Model/RuntimeResponseGeneric.cs:160]: need to remove this line

**Pull request merged**